### PR TITLE
GNU tar 1.30

### PR DESCRIPTION
**Pull request by Mno-hime**
`03-CVE-2016-6321.patch` was upstreamed. Two new upstream patches for tests.

```
156: sparse files with unicode names                 FAILED (sparse07.at:24)
175: remove-files with compression                   FAILED (remfiles01.at:32)
176: remove-files with compression: grand-child      FAILED (remfiles02.at:32)
```

### CODE REVIEW

**Comment by Mno-hime** [comment]
> 156: sparse files with unicode names                 FAILED (sparse07.at:24)
```
< \305\276lu\305\245
---
> ?lu?
156. sparse07.at:21: 156. sparse files with unicode names (sparse07.at:21): FAILED (sparse07.at:24)
```

> 175: remove-files with compression                   FAILED (remfiles01.at:32)
> 176: remove-files with compression: grand-child      FAILED (remfiles02.at:32)

Expected (1.29):
```
tar (child): a: Cannot open: Is a directory
tar (child): Error is not recoverable: exiting now
gtar: Child returned status 2
gtar: Error is not recoverable: exiting now
```
Got (1.30):
```
gtar: a: Cannot open: Is a directory
gtar: Error is not recoverable: exiting now
gtar: Child returned status 2
gtar: Error is not recoverable: exiting now
```

GNU tar 1.29's `set_program_name` is able to change `argv[0]`.

**Comment by pyhalov** [comment]
Please, deliver only 64-bit version

File: components/archiver/gtar/gtar.p5m [hunk 1]
```diff
@@ -38,7 +30,6 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 hardlink path=usr/gnu/bin/tar target=../../lib/isaexec pkg.linted.userland.action002.0=true
```
Comment (alarcher) [inline components/archiver/gtar/gtar.p5m]: Do not use isaexec anymore since we deliver only 64bit.

File: components/archiver/gtar/gtar.p5m [hunk 2]
```diff
@@ -1,24 +1,16 @@
...
-# information: Portions Copyright [yyyy] [name of copyright owner]
-#
-# CDDL HEADER END
+
 #
-# Copyright (c) 2011, 2015, Oracle and/or its affiliates. All rights reserved.
+# Copyright 2017 <contributor>
 #
 
 <transform file path=usr.*/man/.+ -> default mangler.man.stability volatile>
```
Comment (alarcher) [inline components/archiver/gtar/gtar.p5m]: Component metadata can be moved to Makefile.

Comment (alarcher) [inline components/archiver/gtar/gtar.p5m]: The manpage transform can be removed.

File: components/archiver/gtar/gtar.p5m [hunk 3]
```diff
@@ -1,24 +1,16 @@
...
-# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
-# If applicable, add the following below this CDDL HEADER, with the
-# fields enclosed by brackets "[]" replaced with your own identifying
-# information: Portions Copyright [yyyy] [name of copyright owner]
-#
-# CDDL HEADER END
+
 #
-# Copyright (c) 2011, 2015, Oracle and/or its affiliates. All rights reserved.
+# Copyright 2017 <contributor>
```
Comment (alarcher) [inline components/archiver/gtar/gtar.p5m]: Your name should go here :)

File: components/archiver/gtar/manifests/sample-manifest.p5m [hunk 4]
```diff
@@ -10,7 +10,8 @@
 #
 
 #
-# Copyright 2016 <contributor>
```
Comment (alarcher) [inline components/archiver/gtar/manifests/sample-manifest.p5m:13]: You copied the Oracle copyright to the sample manifest by mistake.

File: components/archiver/gtar/Makefile [hunk 5]
```diff
@@ -23,12 +23,12 @@
 include ../../../make-rules/shared-macros.mk
```
Comment (alarcher) [inline components/archiver/gtar/Makefile:26]: Add PREFERRED_BITS=64 to install in /usr/bin

**Pull request merged**